src/main/webapp/js/events-suammry-loader.js

```diff
@@ -30,18 +30,28 @@
     timeDiv.classList.add('right-align');
     timeDiv.appendChild(document.createTextNode(new Date(event.timestamp)));
    
+    const idDiv = document.createElement('div');
+    idDiv.classList.add('right-align');
+    idDiv.appendChild(document.createTextNode(event.id));
```

Comment:
i think the event.id is giving a very long and weird number that the users don't need to see. I could add the event using its name.